### PR TITLE
Added Yandex

### PR DESCRIPTION
**Pull request by JignasP**
Yandex search had been added

### CODE REVIEW

File: _data/search-engines.yml [hunk 1]
```diff
@@ -13,8 +13,11 @@
   image: ecosia.svg
   url: https://www.ecosia.org/search
 
-# Removed Givero Search as it is discontinued
```
Comment (hellcp) [inline _data/search-engines.yml:16]: I would keep that note

File: assets/search/Yandex.svg [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```
Comment (hellcp) [inline assets/search/Yandex.svg:2]: Would you export the svg as optimized svg in inkscape?

File: assets/search/Yandex.svg [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+<svg
+   width="50mm"
```
Comment (hellcp) [inline assets/search/Yandex.svg:5]: I would rather have the svg the same size as the others in this directory

**Review by hellcp — CHANGES_REQUESTED** [review]

**Comment by JignasP** [comment]
Surely I will make the changes.

**Pull request closed without merging**